### ISSUE TITLE
add F-P link between GO:0016926 protein desumoylation and GO:0016929	SUMO-specific protease activity

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#13941]
GO:0016926 protein desumoylation 
 --part_of SUMO-specific protease activity

**Comment by pgaudet** [comment]
really ?

**Comment by ValWood** [comment]
SUMO-specific protease activity is part of  GO:0016926 protein desumoylation

(it removes the SUMO from proteins)?

I would say its the ONLY part_of GO:0016926 protein desumoylation because it's a single step process.

**Comment by ValWood** [comment]
As I was curating some SUMO protease papers I noticed there is a slight problem with these terms.

SUMO-specific protease activity
defined
Catalysis of the hydrolysis of SUMO, a small ubiquitin-related modifier, from previously sumoylated substrates. 

This is correct,  when people refer to SUMO-specific protease activity, they mena the activity that cleaves sumo conjugates from proteins.

However this term has 2 children.

GO:0070139 Name | SUMO-specific endopeptidase activity
Catalysis of the hydrolysis of peptide bonds between an alpha-carboxyl group and an alpha-amino group within the small conjugating protein SUMO.

This is describing the  C terminal cleavage of SUMO itself, and is a maturation step of SUMO.
I don't think 
![sumo protease](https://user-images.githubusercontent.com/7359272/28389473-2a406cc8-6cce-11e7-9986-6d46c8a1dfea.jpg)

this should be represented as a type of  "SUMO-specific protease activity"
because it is a different context.

I captured the difference between the 2 as follows:

The other child

GO:0070140
--
Name | SUMO-specific isopeptidase activity
Ontology | Molecular Function
Definition | Catalysis of the hydrolysis of an isopeptide bond between the small conjugating protein SUMO and a protein to which SUMO has been conjugated.

is equivalent to the parent.

I think  GO:0070139 Name | SUMO-specific endopeptidase activity
should be obsoleted, becasue it doesn't fit the parent definition (it isn't deconjugating SUMO), and it's a gene product specific cleavage event (should be captured with a generic protease has_substrate SUMO).

(7 experimental annotation 6 reactome, 1 TAIR)

**Comment by ukemi** [comment]
I think the issue here lies in the definition of the parent term. It should be defined like its parent, GO:0019783--Catalysis of the hydrolysis of peptide or isopeptide bonds within small proteins such as ubiquitin or ubiquitin-like proteins (e.g. APG8, ISG15, NEDD8, SUMO), or between the small protein and a larger protein to which it has been conjugated. Source: GOC:ma, GOC:mah

I suggest we loosen the definition of GO:0016929 from--Catalysis of the hydrolysis of SUMO, a small ubiquitin-related modifier, from previously sumoylated substrates.

to--Catalysis of the hydrolysis of peptide or isopeptide bonds within SUMO) or between SUMO and a larger protein to which it has been conjugated.

@mah11  many of these these terms have your xref, do you recall their original intent?

**Comment by mah11** [comment]
> @mah11 many of these these terms have your xref, do you recall their original intent?

Sorry, no, I don't recall much about this branch, and I suspect I didn't do much besides plug text definitions in. It looks like the "[small thing]-specific protease activity" terms all follow the same definition pattern, and I don't remember any conscious effort to do otherwise.

The whole branch of MF terms for peptidase activities affecting small Ub-like modifiers looks a bit dodgy, which I strongly doubt was intentional. Broadening the definition of GO:0016929 as you suggest will help, and the same should be done for the sibling terms for other small modifying proteins.

But I also think there's a problem with the parts of the definitions that mention which bonds can be cleaved. I don't remember whether isopeptide bond cleavage should fit the definition of "peptidase activity", but at present the Ub-like yada terms are the only ones that mention anything other than peptide bonds. And I'm reasonably confident that ester, thioester, and amide bond cleavage do _not_ fit into the scope of "hydrolysis of peptide bonds", so the GO:0036459 is_a GO:0019783 (which in turn is_a GO:0008234, and so on) is a howler of a TPV. I'm not even sure "thiol-dependent" fits under "cysteine-type peptidase". I hope I didn't create that mess!

**Comment by ukemi** [comment]
Thanks @mah11 

I don't think it is as bad as it might seem. I assume that the peptidase working group organized the top levels of the ontology correctly and isopeptidase is a type of peptidase. So with the defintion tweaks that I suggested to all of the Ub-like yada terms, things should be in order at least for those. Then the children would fall into place with respect to being appropriate subtypes of the parent.

Looking at the EC link from GO:0036459 suggests that the thiol-dependent paths are ok.

**Comment by ValWood** [comment]
I still think its odd to combine  removal of a protein modifier from a protein with the processing(maturation)  of the modifier.  The first activity is part of desumoylation and  the second is part of protein maturation (although its a particular protein maturation that is required  for "sumoylation"  and indirectly  "desumoylation" to occur in the first place.

I can see how these were lumped historically (and this is old GOC:ma),  because the same enzyme can perform the  deconjugation step, and the processing step.

When biologist refer to SUMO specific proteases, they are referring to SUMO isopeptidases.

For example:
https://genomebiology.biomedcentral.com/articles/10.1186/s13059-014-0422-2
SUMOylation is a highly dynamic process, where deconjugation (deSUMOylation) is catalyzed by a family of cysteine proteases, termed SUMO-specific proteases or SUMO isopeptidases. A subset of these enzymes also functions in processing of the SUMO precursor proteins, which is a prerequisite for their conjugation. In this review, we summarize the current view of SUMO deconjugating enzymes.


I really think we need to  remove the grouping term SUMO-specific proteases  because it is just a grouping term for 2 activities often performed by the same enzyme.
SUMO-specific protease  should then be an exact synonym for  "SUMO isopeptidase"

SUMO isopeptidase can have the f-p links to deconjugation.

The processing of SUMO itself is a gene product specific proteolysis activity (not a process in itself like desumoylation of multiple target proteins),  it's gene product specific so it should not  really have a term SUMO-specific endopeptidase activity in modern GO. It can easily be captured with a "endopeptidase" has_substrate SUMO
annotation.

**Comment by ValWood** [comment]
PMID:28284030 actually calls both SUMO-specific proteases (so its more of a "gene product name". But I still think  my argument for not having a grouping term stands. Different activities, same gene product "SUMO specific protease". In this case, I think "SUMO specific protease" should be a related synonym of both terms rather than a parent. The grouping term should be obsoleted and people should reannotate to the isopeptidase, the endopeptidase or both depending on what is appropriate for their data. There is no reason to make direct annotation to the  existing grouping term. It is only making the annotation less precise than it could be.

**Comment by mah11** [comment]
some replies to David ...

> I don't think it is as bad as it might seem. I assume that the peptidase working group organized the top levels of the ontology correctly and isopeptidase is a type of peptidase.

I was in the peptidase working group, and I do remember that work fairly well. We didn't consider isopeptide bonds explicitly, but I've had another look now and learned that a peptide and isopeptide bonds are chemically the same bond, and the difference is where they are in proteins. So now I think distinguishing them would fall under the aim to get rid of substrate specificity in the peptidase branch, except where annotators clamored for, well, exceptions. Ubiquitin and similar modifiers were granted exceptions, so I don't have a problem with continuing to do so.

tl;dr: sure, go ahead as you suggest

> Looking at the EC link from GO:0036459 suggests that the thiol-dependent paths are ok.

One thing I remember particularly clearly from the peptidase work is that Neil and Alan both regarded the EC peptidase classification as unreliable, and they actively encouraged us to make GO MF diverge from EC specifically in the peptidase branch.

My problem is that GO has defined the root-most "peptidase activity" term (GO:0008233) as "Catalysis of the hydrolysis of a peptide bond." Ester and thioester bonds are not peptide bonds.

GO:0036459 post-dates the peptidase work by some years and may have been unintentionally misplaced.

(I'm neutral on the question of grouping vs. not grouping "any peptidase that has anything to do with Ub/SUMO/othersmallthing"; any of 'em violate the substrate-specificity purge, but in a way I've been willing to tolerate.)

**Comment by ukemi** [comment]
I still think the gouping is ok and annotators should simply annotate to the most specific term possible.

As far as the divergence from EC, I don't really have the expertise to do that without a great deal of research and help from experts, or at least the people who were in the peptidase working group. To close this ticket in a timely manner, I suggest that I make the tweaks to the definitions that I suggested above and solve the Ub-like blah issue. If we want to rework the peptidases again, we should open another ticket and tag it as a miniproject..

**Comment by ukemi** [comment]
> I really think we need to remove the grouping term SUMO-specific proteases because it is just a grouping term for 2 activities often performed by the same enzyme.
SUMO-specific protease should then be an exact synonym for "SUMO isopeptidase"

No, it's a grouping term based on similar molecular functions/biochemical activities.

**Comment by ValWood** [comment]
OK good plan, too much work for a small gain. I will make a note to move our modifiers down to the specific terms instead. A good compromise would also be to flag as not for direct annotation. I will do this for us internally.

**Comment by ValWood** [comment]
> No, it's a grouping term based on similar molecular functions/biochemical activities.

Doesn't the commonality has more to do with the target molecule  but in different context ?

**Comment by ukemi** [comment]
I think the flag is a good idea! Let me check existing annotations. We can bring this up on an annotation call and allow people to move their annotations as well. I suspect that a lot of people annotated just based on the term name and never investigated the children. @vanaukenk  can we add this to an agenda for a call?

**Comment by ValWood** [comment]
yeah ignore me, they are both isopeptide bonds ...

**Comment by ukemi** [comment]
PS> I think the flag is only a good idea when we haven't enumerated the possible children. It looks like they don't exist for all the terms. I'm not sure if all of these molecules are process in the same manner, do you know?
Maybe that's why some of the Ub-like blah protease terms don't have children and are defined more precisely. Do you know off the top of your head?

**Comment by ValWood** [comment]
This might be the case. Ubiquitin is clearly processes but thats because it is  sometimes transcribed as tandem copies. I don't know if we know what does the processing (I'm sure we must). For the other modifiers I don't think they have been studied in so much depth that we know about any processing....

I have a vague inckling that I might have requested the parentage for the SUMO term in the past. I really thought I had, but then changed my mind when I looked and we had not annotated to the children....

But I bet this is the reason that this structure only exists for SUO....

**Comment by ukemi** [comment]
But the original parent term is defined as though there are two flavors, so I suspect it wasn't just you. Maybe the best thing for me to do now is stick to the SUMO term and any others that have two flavors. I will leave the others as they are until we find that they also use the processing enzymes.

**Comment by deustp01** [comment]
Val said, "Ubiquitin is clearly processed but that's because it is sometimes transcribed as tandem copies." In humans, it's always transcribed as a tandem polyprotein, sometimes poly-Ub and sometimes a ribosomal protein and Ub. A recent review argues that the same deubiquitinases responsible for cleaving the isopeptide bonds that attach ubiquitin to target proteins, also cleave the peptide bonds in the tandem proteins to release ubiquitin monomers (PMID: 26235645). The same group also appears to argue (guessing from the abstract) that there is a similar overlap for SUMO processing and de-SUMOylation (PMID: 26522917).

**Comment by vanaukenk** [comment]
@ukemi  - Yes, we can certainly add this item to an annotation call, if needed.  The next call is not scheduled until August 8th, though.

**Comment by vanaukenk** [comment]
For today's annotation call, these are the existing experimental annotations in AmiGO:

**GO:0016929 SUMO-specific protease activity**

- uspl1, ULP1D, ESD4, OTS2 - UniProtKB

- SENP2, senp2 - UniProt

- ULP1A, ULP1D, ESD4, OTS2 - TAIR

- ULP1, ULP2 - SGD

- Ulp1 - FlyBase

- PF3D7_1233900, SENP1 - GeneDB

- Senp1, Senp2 - MGI

- ulp1 - PomBase

**GO:0070139 SUMO-specific endopeptidase activity**

- SENP1, SENP2, SENP5 - Reactome

- ASP1 - TAIR

**GO:0070140 SUMO-specific isopeptidase activity**

- ULP1, ULP2, ULP3 - CGD

- Uspl1 - MGI

- Uspl1 - RGD

- WSS1 - SGD (a NOT annotation)

- USPL1 - UniProt

- uspl1 - UniProtKB
